src/config/config.ts

```diff
@@ -325,6 +328,11 @@ export class Config implements IConfig {
 
     this.npmRegistry = this.scopedEnvVar('NPM_REGISTRY') || this.pjson.oclif.npmRegistry
 
+    const themeFilePath = resolve(this.configDir, 'theme.json')
+    const theme = this.scopedEnvVarTrue('DISABLE_THEME') ? undefined : await safeReadJson(themeFilePath)
```

Comment:
```typescript
const theme = this.scopedEnvVarTrue('DISABLE_THEME') ? undefined : await safeReadJson<Record<string, string>>(themeFilePath)
```

That way you don't have to use `as` when passing it to `parseTheme`

Reply:
fixed at 00a6ccdde08f13b44eec55b57a6d6e584f02d9fd

Could you tell me if you agree with what I did. I removed enableTheme from config.